README.md

```diff
@@ -74,11 +74,10 @@ NOTIFY_TEMPLATE_ID=d72e2ff9-b228-4f16-9099-fd9d411c0334
 The service uses GOV.UK Verify to verify the identity of teachers that are
 claiming.
 
-If you want to test Verify in development, add the following to your `.env`
-file.
+In order to integrate with Verify in development, add the following to your
+`.env` file.
 
 ```
-GOVUK_VERIFY_ENABLED=1
 GOVUK_VERIFY_VSP_HOST=http://localhost:50300
```

Comment:
Given this isn't really configurable or optional, maybe we should create a `.env.development` again instead of getting developers to maintain their own config?